docs/guides/credentials.rst

```diff
@@ -6,57 +6,117 @@ Credentials
 The Application Framework implements OAuth 2.0 for granting authorization to
 access your Logging, Event and Directory-Sync instances.
 
-In the near future, ``pancloud`` will come packaged with OAuth2 support to
-help ease the process of:
+``pancloud`` comes packaged with OAuth 2.0 support to ease the process of:
 
-   - Performing OAuth2 and fetching tokens
+   - Generating authorization URL and fetching tokens
    - Refreshing tokens
    - Revoking tokens
+   - Auto token refresh and auto-retry
 
 Obtaining and Using Tokens
 --------------------------
 
-Work with your Developer Relations representative to deploy and register an
-API Explorer instance, which can be used to retrieve tokens.
+Work with your Developer Relations representative to register your
+application and receive the credentials needed to obtain an ``ACCESS_TOKEN``.
```

Comment:
I think access_token, client_id, client_secret and refresh_token should be in lower case when referring to the value/id, and only upper case when referring to the environment variable.  other cases to modify are below.